infra/resources/src/main/res/values-am/strings.xml

```diff
@@ -429,4 +433,28 @@
     <!-- Notifications -->
     <string name="notification_sync_description">ሲንክ በማድረግ ላይ</string>
 
+    <!-- Identification pool validation -->
+    <string name="id_pool_validation_issue_title">የመለየት ችግር</string>
+    <string name="id_pool_validation_default_message">የልየታ እጩዎችን በማጣራት ላይ</string>
+    <string name="id_pool_validation_user_mismatch_message">የተጠቃሚው መለያ እና ሲንከ የተደረገው መረጃ አልተመሳሰለም</string>
+    <string name="id_pool_validation_module_mismatch_message">ሞጁሉ መለያ ሲንክ ከተደረገው መረጃ ጋር አልተመሳሰለም</string>
+    <string name="id_pool_validation_sync_required_message">መረጃው በቅርቡ አልተላከም</string>
+    <string name="id_pool_validation_syncing_message">ሲንክ በማድረግ ላይ</string>
+    <string name="id_pool_validation_pool_empty_message">ስልኩ ላይ ምንም መረጃ የለም</string>
+    <string name="id_pool_validation_action_close">ዝጋ</string>
+    <string name="id_pool_validation_action_sync">ሲንከ አድርግና እና እንደገና ሞክር</string>
+
+    <!-- Age group selection -->
+    <string name="age_group_selection_title">የተጠቃሚውን የእድሜ ክልል ምረጥ</string>
+    <string name="age_group_selection_age_range_to">ወደ</string>
+    <string name="age_group_selection_age_range_and_above">እና በላይ</string>
+    <plurals name="age_group_selection_age_in_months">
+        <item quantity="one">%d month</item>
+        <item quantity="other">%d months</item>
+    </plurals>
+    <plurals name="age_group_selection_age_in_years">
```

Comment:
`age_group_selection_age_in_months` and `age_group_selection_age_in_years `should be translated too

Reply:
Will be sorted later